bundles/org.eclipse.passage.lic.base/src/org/eclipse/passage/lic/base/io/Settings.java

```diff
@@ -44,6 +39,8 @@
  */
 public final class Settings {
 
+	private static final Predicate<Path> IS_SETTINGS_FILE = new PassageFileExtension.Settings()::ends;
```

Comment:
no need for this to be a field, it is a method

Reply:
Done. Moved it down into the extracted method.